WebUtils/src/org/labkey/webutils/view/knockout_components/lk-table.jsp

```diff
@@ -70,7 +70,8 @@
             <!-- ko foreach2: {data: $parent.table.rows } -->
             <tr data-bind="css: { 'clickable': $component.rowsAreClickable, 'warning': warn, 'danger': err}, visible: !isHidden(), style: {background-color: isSelected() ? $component.rowBackgroundColorClicked : '', 'cursor' : $component.cursor} ">
                 <!-- ko if: $component.rowsAreSelectable -->
-                <td onclick="event.stopPropagation();"> <%-- prevent click from propagating to the row. --%>
+                <td id="chk-btn"> <%-- prevent click from propagating to the row. --%>
+                    <% addHandler("chk-btn", "click", "event.stopPropagation()"); %>
```

Comment:
I don't know anything about "knockout", but it looks like this is maybe iterating through rows of `parent.table.rows`... have you tested this?

Reply:
I think I tested this and it was working fine.